README.md

```diff
@@ -66,8 +66,31 @@ When you have implemented the tool, answer the questions below, commit it to Git
 
 How does your method for extracting features work?
 
+Firstly I make sure the format is "ideal" bed with tabs seperating the 4 couloums. Accomplishing this i use
+    parse_line function, in a for loop over the input file, to parse each line of the bedfile.
+Then I define every value into their position (abusing the fact that we are working with input no larger than 3).
+    This is done through naming every position making them alle easily accesible for searching through.
+Afterwards every line that is on the chromosome of interest is added to a list, basicly creating a subset of the
+    input file. This is accomplished through the "get_chrom" command.
+Lastly I run through this list checking if a line fits in the designated area, the user has searched for.
+    If the start and end is found within that searched area it gets printed directly to the output.file.
+    This one is accomplished through a for loop, of every line/entry in the list that was made in the earlier step.
+    Then checking if the start and end positions are within the seared interval by an if statement at the end of the loop.
+
 What is the complexity of the algorithm, as a function of the size of the two input files? When you answer this, you need to know that you can get the list of chromosomse from a `query.Table` in constant time, but it does, of course, take longer to run through all the lines in it.
 
+Not-complex? I have avoided any heavy processing by making the code only function for reagion no larger nor smaller than 3.
+    So for the most part i would assume its constant in that i am rather specific with telling it to look at position X, rather
+    than searching for something more general making sure to avoide pitfalls like overlaps.
```

Comment:
"Not complex" is not really a description we use when we have a formal notation for what we are doing ;)

I agree that the code is far from complex, but it is the time complexity I ask about. If we say that there are $n$ features in the bed file and $m$ queries, what is the asymptotic running time in $O(-)$-notation?

And what do you mean when you say regions are no larger or smaller than 3? Do you mean three nucleotides? Because that isn't the case for the test data, where all bed-file features have a length of 1 and the query regions have variable size.

Reply:
mb forgot what complexity meant in the context of an algorithm. Have pushed my corrections but just incase u want the correction here:
OqB (or with the annotations you used here; Omn)
q being the length of the query and then B the size of the BED file.

Reply:
Looks fine.